### PR TITLE
docs: Add CoC and git parts from Kinvolk's contribution repository

### PR DESCRIPTION
**Pull request by joaquimrocha**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -11,6 +12,10 @@
 - [Updating dependencies](#updating-dependencies)
 - [Testing and linting requirements](#testing-and-linting-requirements)
 
+## Code of Conduct
+
+Please refer to the Kinvolk [Code of Conduct](https://github.com/kinvolk/contribution/blob/master/CODE_OF_CONDUCT.md).
```
Comment (invidian) [inline docs/CONTRIBUTING.md:17]: I wonder if this will be picked up by https://github.com/kinvolk/lokomotive/community.

**Review by invidian — APPROVED** [review]
LGTM, just one Q

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (joaquimrocha) [reply docs/CONTRIBUTING.md:17]: Interesting. Probably it won't. But if that's the case we could add a `CODE_OF_CONDUCT.md` file just with the same text, like the Kubernetes Dashboard does: https://github.com/kubernetes/dashboard/blob/master/code-of-conduct.md

In either case, I think it's important to keep this section in the `contributing` doc.

**Review by ipochi — APPROVED** [review]
LGTM

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (invidian) [reply docs/CONTRIBUTING.md:17]: I'd add this file and keep the section then. We will need to periodically update it then.

Comment (vbatts) [reply docs/CONTRIBUTING.md:17]: I see an "add" button on that community page. I think we can link it once merged

Comment (vbatts) [reply docs/CONTRIBUTING.md:17]: hm. maybe

**Comment by vbatts** [comment]
LGTM

**Pull request merged**